generator/generator_config.textproto

```diff
@@ -482,6 +482,71 @@ service {
   retryable_status_codes: ["kUnavailable"]
 }
 
+
+# BigQueryControl
+service {
+  service_proto_path: "google/cloud/bigquery/v2/dataset.proto"
+  product_path: "google/cloud/bigquerycontrol/v2/dataset"
```

Comment:
I was suggesting: `product_path: google/cloud/bigquerycontrol/v2` for all of these